.scrutinizer.yml

```diff
@@ -126,3 +126,19 @@ build_failure_conditions:
 
     # Don't let Code Quality Rating drop below 6.
     - 'project.metric("scrutinizer.quality", < 6)'
+
+build:
```

Comment:
For what it's worth, I don't think we've ever used Scrutinizer for running our tests before. Our typical practice is using Codeship.